utils/backend/mysk/fetchMySKProxy.ts

```diff
@@ -0,0 +1,31 @@
+import { FetchReturn, Query } from "@/utils/types/fetch";
+
+/**
+ * Fetches data via the MySK API proxy.
+ *
+ * @param path The path to make the fetch request to.
+ * @param options `fetch` options and query parameters.
+ *
+ * @returns The response from the API.
+ *
+ * @private Use `useMySKClient` instead.
+ */
+export default async function fetchMySKProxy<
+  Data extends {} | unknown = unknown,
+>(
+  path: string,
+  options?: Partial<RequestInit & { query: Query }>,
+): Promise<FetchReturn<Data>> {
+  // Fetch the data via the MySK API proxy
+  const response = await fetch("/api/mysk-api-proxy", {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify({ path, options }),
+  });
```

Comment:
I want your opinion on this. Is it bad that all fetches are _technically_ POST? Does it matter?

Reply:
Okie!